src/main/java/gregtech/loaders/recipe/chemistry/TitaniumRecipes.java

```diff
@@ -0,0 +1,157 @@
+                .input(dust, CalciumChloride, 6)
+                .fluidInputs(Water.getFluid(4000))
+                .fluidOutputs(BauxiteSlurry.getFluid(4000))
+                .duration(500).EUt(VA[HV]).buildAndRegister();
+
+        // Bauxite (washed) + Soda Ash + Calcium Chloride -> Bauxite Slurry
+        MIXER_RECIPES.recipeBuilder()
+                .input(crushedPurified, Bauxite, 32)
+                .input(dust, SodaAsh, 12)
+                .input(dust, CalciumChloride, 6)
+                .fluidInputs(Water.getFluid(4000))
+                .fluidOutputs(BauxiteSlurry.getFluid(4000))
+                .duration(500).EUt(VA[HV]).buildAndRegister();
+
+        // Bauxite Slurry -> Cracked Bauxite Slurry
+        CRACKING_RECIPES.recipeBuilder()
+                .circuitMeta(1)
+                .fluidInputs(BauxiteSlurry.getFluid(16000))
+                .fluidInputs(Steam.getFluid(1000))
+                .fluidOutputs(CrackedBauxiteSlurry.getFluid(16000))
+                .duration(500).EUt(VA[HV]).buildAndRegister();
+
+        // Bauxite Slurry + Sulfuric -> Aluminium, Slag, Sludge, and SO3 (for looping back to Sulfuric Acid)
+        LARGE_CHEMICAL_RECIPES.recipeBuilder()
+                .fluidInputs(CrackedBauxiteSlurry.getFluid(4000))
+                .fluidInputs(SulfuricAcid.getFluid(1000))
+                .output(dust, Aluminium, 24)
+                .output(dust, BauxiteSlag, 8)
+                .fluidOutputs(BauxiteSludge.getFluid(2500))
+                .fluidOutputs(SulfurTrioxide.getFluid(1000))
+                .duration(500).EUt(VA[HV]).buildAndRegister();
+
+        // Bauxite Slag -> Salt (looped) + Nd + Cr (byproducts)
+        ELECTROMAGNETIC_SEPARATOR_RECIPES.recipeBuilder()
+                .input(dust, BauxiteSlag)
+                .output(dust, Salt)
+                .chancedOutput(dust, Neodymium, 2000, 250)
+                .chancedOutput(dust, Chrome, 1000, 250)
+                .duration(50).EUt(VA[MV]).buildAndRegister();
+
+        // Bauxite Sludge -> Calcite (looped) + Decalcified Bauxite Sludge
+        DISTILLERY_RECIPES.recipeBuilder()
+                .circuitMeta(1)
+                .fluidInputs(BauxiteSludge.getFluid(500))
+                .output(dust, Calcite, 2)
+                .fluidOutputs(DecalcifiedBauxiteSludge.getFluid(500))
+                .duration(100).EUt(VA[MV]).buildAndRegister();
+
+        // Decalcified Bauxite Sludge -> Rutile, Gallium, SiO2, Iron, Water
+        CENTRIFUGE_RECIPES.recipeBuilder()
+                .fluidInputs(DecalcifiedBauxiteSludge.getFluid(250))
+                .output(dust, Rutile, 2)
+                .chancedOutput(dust, Gallium, 5000, 550)
+                .chancedOutput(dust, Gallium, 3000, 800)
+                .chancedOutput(dust, Gallium, 1000, 1000)
+                .chancedOutput(dust, SiliconDioxide, 9000, 250)
+                .chancedOutput(dust, Iron, 8000, 250)
+                .fluidOutputs(Water.getFluid(250))
```

Comment:
This may be net positive on water. Not sure if intentional.

Reply:
It shouldn't be, every step in the chain is 1:1 water:
- 4kL water -> 4kL slurry
- 16kL slurry -> 16kL cracked slurry
- **4kL cracked slurry -> 2.5kL sludge**
- 500L sludge -> 500L decalcified sludge
- 250L decalcified sludge -> 250L water

So some water is lost because of separation of cracked slurry into sludge